### PR TITLE
Syncthru Printer description

### PR DESCRIPTION
**Pull request by nielstron**
**Description:**
Adding a samsung syncthru printer component

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#13134

## Checklist:

- [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.syncthru.markdown [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+layout: page
+title: "Samsung SyncThru Printer"
+description: "Instructions how to integrate a Samsung printer providing SyncThru within Home Assistant."
+date: 2018-02-19 23:33
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: # where do I place this
```
Comment (frenck) [inline source/_components/sensor.syncthru.markdown]: `/source/images/supported_brands`

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_components/sensor.syncthru.markdown (hunk 1 above)
Comment (nielstron) [reply source/_components/sensor.syncthru.markdown]: How do you handle copyright issues? Where to store information that we are allowed to use brand logos?

**Review by frenck — APPROVED** [review]

File: source/_components/sensor.syncthru.markdown [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    required: false
+    default: Samsung Printer
+    type: string
+  monitored_conditions:
+    description: Conditions to display in the frontend.
+    required: false
+    default: all values
+    type: list
+    keys:
+      toner_black:
```
Comment (frenck) [inline source/_components/sensor.syncthru.markdown:51]: The descriptions for each of these keys are missing.
@nielstron Could you please add them?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_components/sensor.syncthru.markdown (hunk 2 above)
Comment (nielstron) [reply source/_components/sensor.syncthru.markdown:51]: Added descriptions

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @nielstron!

I'll go ahead and merge this one in, since the parent PR has been merged already.

**Pull request merged**